babynames.py

```diff
@@ -45,9 +45,30 @@ def extract_names(filename):
     with the year string followed by the name-rank strings in alphabetical order.
     ['2006', 'Aaliyah 91', Aaron 57', 'Abagail 895', ' ...]
     """
-    names = []
+    with open(filename) as f:
+        open_file = f.read()
```

Comment:
Here you want to use a variable name that represents the data.  Instead of 'open_file' I would name it 'html_text' because that's what it contains now